mill-tpolecat/src/io/github/davidgregory084/TpolecatModule.scala

```diff
@@ -5,6 +5,6 @@ import mill.scalalib._
 
 trait TpolecatModule extends ScalaModule {
   override def scalacOptions = T {
-    Tpolecat.scalacOptionsFor(scalaVersion())
+    super.scalacOptions() ++ Tpolecat.scalacOptionsFor(scalaVersion())
```

Comment:
The other of the traits might mean - I think - that you could accidentally remove other settings previously set, if you don't append. But I could be wrong. This change is a separate commit and easy to revert.